deeppavlov/models/morpho_tagger/README.md

```diff
@@ -0,0 +1,19 @@
+It is an implementation of neural morphological tagger from
+[Heigold et al., 2017. An extensive empirical evaluation of character-based morphological tagging for 14 languages](http://www.aclweb.org/anthology/E17-1048).
+We distribute the model trained on ru_syntagrus corpus of [Universal Dependencies project](www.universaldependencies.org).
+If you want to use it from scratch, do the following:
+
+1. Download the archive from 
+<http://lnsigo.mipt.ru/export/deeppavlov_data/morpho_tagger.tar.gz> to
+ [../../../download](../../../download).
```

Comment:
There is no need to download anything by hands, there are commands for that

Reply:
And this link gives 404

Reply:
I will fix it.

Reply:
You mean the link to download subdirectory?

Reply:
yes. If you open this file in github